tests/blockchain/eth/entities/agents/test_staking_escrow_agent.py

```diff
@@ -224,6 +224,35 @@ def test_divide_stake(agency, token_economics):
     assert len(stakes) == 3
 
 
+@pytest.mark.slow()
+def test_enable_restaking(agency, testerchain):
+    token_agent, staking_agent, _policy_agent = agency
+    staker_account, worker_account, *other = testerchain.unassigned_accounts
+    receipt = staking_agent.set_restaking(staker_account, value=True)
+    assert receipt['status'] == 1, "Transaction Rejected"
+
+
+@pytest.mark.slow()
+def test_lock_restaking(agency, testerchain):
+    token_agent, staking_agent, _policy_agent = agency
+    staker_account, worker_account, *other = testerchain.unassigned_accounts
+    current_period = staking_agent.get_current_period()
+    terminal_period = current_period + 2
+    receipt = staking_agent.lock_restaking(staker_account, release_period=terminal_period)
+    assert receipt['status'] == 1, "Transaction Rejected"
```

Comment:
Similar as above. Shouldn't we check here that `staking_agent.is_restake_locked(staker_account)`?

Reply:
:heavy_check_mark: